README.md

```diff
@@ -430,11 +439,9 @@ docker run --rm \
     -v $(pwd)/dkim.pub.key:/dkim.pub.key \
     -v $(pwd)/simplelogin.env:/code/.env \
     --network="sl-network" \
-    simplelogin/app:3.4.0 flask db upgrade
+    simplelogin/app:4.35.2 alembic upgrade head
```

Comment:
Use the same tag as above (here and below). Users might forget to replace the version number.